circle.yml

```diff
@@ -12,7 +12,6 @@ dependencies:
     - echo y | $ANDROID_HOME/tools/bin/sdkmanager "platform-tools" "extras;android;m2repository" "extras;google;m2repository"
 test:
   override:
-    - ./gradlew assemble # To build *.aar files
```

Comment:
this task was added by https://github.com/cookpad/license-tools-plugin/pull/84, but it was not necessary in another solution https://github.com/cookpad/license-tools-plugin/pull/85